src/components/modes/teacher/AssignBadgeForm.js

```diff
@@ -7,6 +7,7 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { withNamespaces } from 'react-i18next';
 import { postAppInstanceResource } from '../../../actions';
 import { BADGE } from '../../../config/appInstanceResourceTypes';
+import './AssignedBadges.css';
```

Comment:
Here you should create a new CSS file and call it `AssignBadgeForm.css` containing the rules necessary for this file.